packages/react/src/auto/mui/MUIAutoForm.tsx

```diff
@@ -53,6 +49,9 @@ export const MUIAutoFormComponent = <
   props: MUIAutoFormProps<GivenOptions, SchemaT, ActionFunc>
 ) => {
   const { record: _record, action, findBy, ...rest } = props as MUIAutoFormProps<GivenOptions, SchemaT, ActionFunc> & { findBy: any };
+
+  validateAutoFormProps(props);
+
```

Comment:
This should be moved up to where you removed `if (!action)` just like polaris AutoForm
Otherwise, when `action === undefined` we will get the `can't reference operationName on undefined` before we get here